Knysh/Dijkstra/dj.C

```diff
@@ -0,0 +1,58 @@
+// spiszhenno, no zato razobralsa
+#include <iostream>
+#include <vector>
+#define INT_MAX 100000
+using namespace std;
+
+void Dijkstra(const std::vector<std::vector<int> >& GR, int st)
+{
+  int V = (int)GR.size();
```

Comment:
C-style cast is bad. Change to `static-cast` or `dynamic-cast`.
Give meaningful name for variable.